src/crash.cpp

```diff
@@ -187,7 +195,108 @@ extern "C" {
 
 } // extern "C"
 
-[[noreturn]] static void terminate_hanlder()
+void init_crash_handlers()
+{
+    SymInitialize( GetCurrentProcess(), NULL, TRUE );
+    ULONG stacksize = 2048;
+    SetThreadStackGuarantee( &stacksize );
+    for( auto sig : {
+             SIGSEGV, SIGILL, SIGABRT, SIGFPE
+         } ) {
+
+        std::signal( sig, signal_handler );
+    }
+    std::set_terminate( crash_terminate_handler );
+}
+
+#else
+// Non-Windows implementation
+
+#include <sstream>
+#include "debug.h"
+
+extern "C" {
+
+    static const char *get_crash_log_file_name()
+    {
+        auto crash_it = FILENAMES.find( "crash" );
+        if( crash_it != FILENAMES.end() ) {
+            return crash_it->second.c_str();
+        }
+        return "crash.log";
```

Comment:
If the access to FILENAMES works then it probably will be config/crash.log.  If it doesn't, then I didn't want to assume the existence of the config directory, since it's possible that the current working directory could be just about anywhere.  Will add a comment to that effect.